src/collective/easyform/browser/widgets.py

```diff
@@ -61,3 +64,28 @@ class LabelRenderWidget(ViewMixinForTemplates, BrowserView):
 
 class RichLabelRenderWidget(ViewMixinForTemplates, BrowserView):
     index = ViewPageTemplateFile("rich_label.pt")
+
+
+class IRenderWidget(Interface):
+    """
+    """
+
+
+# overriding plone.app.z3cform widget.pt:
+@implementer(IRenderWidget)
+class RenderWidget(ViewMixinForTemplates, BrowserView):
+    index = ViewPageTemplateFile('widget.pt')
+
+
+@adapter(IRenderWidget, Interface)
+@implementer(IBrowserView)
+class WidgetDependencyView(object):
+    def __init__(self, widget, request):
+        self.widget = widget
+        self.request = request
+
+    def __call__(self):
+        depends_on = self.widget.context.field.queryTaggedValue("depends_on")
+        if not depends_on:
+            return ""
+        return depends_on
```

Comment:
If EasyForm is installed this is available for all of z3c.form, like DX-Forms, too, right? Is this favorable? Or IWO if we add the feature, shouldn't it be moved to plone.app.z3cform then? Could this have side effects on other addons/custom projects fiddeling with widgets?

Reply:
I've updated that so that the custom renderer is only available for `IEasyFormWidget`